ci/do_ci.sh

```diff
@@ -49,6 +49,10 @@ if [[ "$1" == "bazel.release" ]]; then
   echo "bazel release build with tests..."
   bazel_release_binary_build
   echo "Testing..."
+  # We have various test binaries in the test directory such as tools, benchmarks, etc. We
+  # run a build pass to make sure they compile.
+  bazel --batch build ${BAZEL_BUILD_OPTIONS} -c opt //test/...
```

Comment:
should we also build //source/... ?  Actually while we are at it, what about //include/... ?

Reply:
I think we probably have almost everything in `//source` and `//include` already captured implicitly by `//test`, but I agree that it doesn't hurt to add them here, Bazel won't do extra work.